netmanager/src/views/pages/Dashboard/Dashboard.js

```diff
@@ -525,62 +530,12 @@ const Dashboard = (props) => {
 
         <Grid item lg={6} md={6} sm={12} xl={6} xs={12}>
           <Grid item lg={12} sm={12} xl={12} xs={12}>
-            <Map id="rootMapContainerId" />
+          <div className={classes.mapContainer}>
+              <Map/>
+            </div>
           </Grid>
 
-          {/* <Grid container spacing={0} className="MapCardContent">
-            <Grid item lg={2} sm={4} xl={2} xs={12}>
```

Comment:
Just not sure about this huge comment/deletion but all looks good on running!

Reply:
That's the old legend bar that was bottom of the map, was replaced with a smaller legend floating on the map. 
Was initially commented out by me.

Reply:
Okay, awesome.